src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -295,6 +301,9 @@ def get_cluster_events_path(cluster, output_folder):
     return os.path.join(output_folder, f"cluster_{cluster['id']}_events.json")
 
 
+def get_cluster_metadata(output_folder):
+    return os.path.join(output_folder, f"metadata.json")
```

Comment:
a couple of suggestions embedded in one:
```suggestion
def get_cluster_metadata(output_folder: str) -> dict:
    with open(os.path.join(output_folder, "metadata.json"), "rt") as f:
        return json.load(f)
```

Reply:
I would but similar functions in this file return file path...

Reply:
NM, fixed